index.js

```diff
@@ -100,14 +119,14 @@ module.exports = {
     app.use(function(req, res, next) {
       var appConfig = project.config(options.environment);
 
-      var header = appConfig.contentSecurityPolicyHeader;
-      var policyObject = appConfig.contentSecurityPolicy;
-
-      if (!header || !policyObject) {
+      if (!appConfig['ember-cli-content-security-policy'].enabled) {
```

Comment:
This string occurs a few times, could move it into a constant, e.g. `const CONFIG_KEY = 'ember-cli-content-security-policy';`